static/js/components/CourseListWithPopover.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import React from 'react';
+import CourseListItemWithPopover from './CourseListItemWithPopover';
+
```

Comment:
we usually just leave a single blank line above the class definition (here and in the other files too)